packages/app/src/hooks/useExplainQuery.tsx

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { sendQuery } from '@/clickhouse';
+import {
+  ChartConfigWithDateRange,
+  renderChartConfig,
+} from '@/renderChartConfig';
+
+export function useExplainQuery(
+  config: ChartConfigWithDateRange,
+  options?: { enabled?: boolean },
+) {
+  const {
+    data: explainData,
+    isLoading,
+    error,
+  } = useQuery({
```

Comment:
I think for the lint you'll want something like `useQuery<ExplainResult[]>` here or something like that

Reply:
Hmm I wonder if I missed a lint call locally. I thought I ran make ci-lint.

Reply:
I thought it was part of precommit as well, maybe something to ping @wrn14897 about if it's wonky.

Reply:
Nope that's just me being new and dumb. I used `make dev-lint` oops